Formula/re-flex.rb

```diff
@@ -0,0 +1,30 @@
+class ReFlex < Formula
+  desc "The regex-centric, fast and flexible scanner generator for C++"
+  homepage "https://www.genivia.com/doc/reflex/html"
+  url "https://github.com/Genivia/RE-flex/archive/reflex-1.0.12.tar.gz"
+  sha256 "75a1761d9a6fd3fee6454e3b6ec954efbad76ece9b68633a07ada977af4cd300"
+
+  depends_on "boost"
+
+  def install
+    system "./configure",
```

Comment:
Do not skip line here, put first argument on the same line. See `jpeg.rb` for how we usually do it.